packages/credentials/src/issuer.ts

```diff
@@ -5,14 +5,17 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import type { Did, ICType, IClaimContents } from '@kiltprotocol/types'
-
 import { SDKErrors } from '@kiltprotocol/utils'
-import { KiltAttestationProofV1, KiltCredentialV1 } from './V1/index.js'
-import type { UnsignedVc, VerifiableCredential } from './V1/types.js'
-import type { CTypeLoader } from './ctype/index.js'
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this linter exception is disconnected from its original target now, so can probably go?

Reply:
f1a54b26e2d9d9a452962f70c01ee00cf3d6e434